src/components/TransactionFlow/helpers.ts

```diff
@@ -13,6 +23,82 @@ type FieldValue = ValuesType<ISender>;
 
 const preferValueFromSender = (l: FieldValue, r: FieldValue): FieldValue => (isEmpty(r) ? l : r);
 
+interface Props {
+  backStepTitle: string;
+  amount: string;
+  account: StoreAccount;
+
+  flowConfig: ITokenMigrationConfig | IMembershipConfig | IAssetPair;
```

Comment:
Can we use `ITxMultiConfirmProps` as part of this prop construction so it isn't defined in two places?